juju-tutorials/juju-tricks.md

```diff
@@ -0,0 +1,260 @@
+[comment]: <> (Juju Tricks)
+
+## Introduction
+
+Duration: 0:01
+
+This article is compilation of CLI tricks, useful for recovery of juju deployments after OS services' hosts reboot / crash / network splits / etc.
```

Comment:
I'd suggest using commas at the end of the sentence: "after services or hosts reboot, crash, or network components become unreachable."